catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/databases/TableResourceTest.java

```diff
@@ -1023,6 +1024,25 @@ void delete_table_200_ok(TestInfo test) throws HttpResponseException {
     deleteEntity(table.getId(), adminAuthHeaders());
   }
 
+  @Test
+  void delete_put_Table_200(TestInfo test) throws IOException {
+    CreateTable request = create(test).withDatabase(DATABASE.getId()).withDescription("");
+    Table table = createEntity(request, adminAuthHeaders());
+
+    // Delete
+    deleteEntity(table.getId(), adminAuthHeaders());
+
+    ChangeDescription change = getChangeDescription(table.getVersion());
+    change.setFieldsUpdated(
+        Arrays.asList(
+            new FieldChange().withName("deleted").withNewValue(false).withOldValue(true),
+            new FieldChange().withName("description").withNewValue("updatedDescription").withOldValue("")));
+
+    // PUT with updated description
+    updateAndCheckEntity(
+        request.withDescription("updatedDescription"), Response.Status.OK, adminAuthHeaders(), MINOR_UPDATE, change);
+  }
+
```

Comment:
@pmbrull, we can explore moving this to EntityResourceTest. For now let's commit this patch. I can pickup moving it to EntityResourceTest.

Reply:
+1 to moving to base class EntityResourceTest

Reply:
Thanks, both. Actually, I started with putting it in the `EntityResourceTest` but got some issues. I can take another look as I must have missed something :)